torch/csrc/jit/ir.h

```diff
@@ -415,6 +427,7 @@ struct Node : public Attributes<Node> {
   // Result:  %3 = f(%1, %2, %4)
   Value* addInput(Value * node) {
     JIT_ASSERT(graph_ == node->owningGraph());
+    schema_ = nullptr;
```

Comment:
Setting the type of an input node will also invalidate the schema, so there needs to be a change to `setType` as well that traverses uses and clears the schema of each use.